lib/foreman/renderer/scope/macros/host_template.rb

```diff
@@ -121,6 +121,39 @@ def ks_console
             (@port && @baud) ? "console=ttyS#{@port},#{@baud}" : ''
           end
 
+          apipie :method, 'Generates commands for package(s) installation, supported OS families: Redhat, Debian & Suse' do
```

Comment:
This makes me wonder if there is a way to give a longer description. No sure if it exists, but would something like this make sense?
```suggestion
          apipie :method, 'Generates commands for package installation' do
            description 'Supported OS families: Redhat, Debian & Suse'
```

cc @ofedoren

Reply:
I've tried to add methods that could just came up in head of the person who writes the docs, so there are three methods you can use (aliases) to write longer description: `desc`, `description`, `full_description`.